xarray/tests/test_dataarray.py

```diff
@@ -43,6 +43,9 @@
     source_ndarray,
 )
 
+pytestmark = pytest.mark.filterwarnings("error:Mean of empty slice")
+pytestmark = pytest.mark.filterwarnings("error:All-NaN slice encountered")
```

Comment:
I think you can assign to a `list`:
```suggestion
pytestmark = [
    pytest.mark.filterwarnings("error:Mean of empty slice"),
    pytest.mark.filterwarnings("error:All-NaN slice encountered"),
]
```
otherwise the first line may be ignored (not sure though, `pytest` might have used some sort of magic to make that work anyways)

Reply:
Great point, thanks!